frictionless/helpers.py

```diff
@@ -631,3 +632,41 @@ def dicts_to_markdown_table(dicts: List[dict], **kwargs) -> str:
         error = errors.GeneralError(note="Please install `pandas` package")
         raise module.FrictionlessException(error)
     return df.where(df.notnull(), None).to_markdown(index=False)
+
+
+def validation_summary(
```

Comment:
I think it's better to have in the `program.validate` module to be close to the place where it's used. We can use a simple logic for functions like this -- in how many places this function is used? if in only one place, it's a good change we don't need to move it into helpers as helpers is a reusable generic library

Reply:
Sure! I have made the changes accordingly and will follow the logic for new changes